crates/bevy_app/src/app.rs

```diff
@@ -114,6 +119,33 @@ impl App {
         }
     }
 
+    /// Creates a new [`App`] with some default structure with all default stages set to use single threaded system executor.
+    pub fn single_threaded() -> App {
```

Comment:
I think ideally this is an "app setting", which we use for all app stages (by default).  Ex: replace instances of `add_stage(Label, stage_reference)` with `add_default_stage(Label)`. And with stageless on the horizon, this would need to be a global app setting anyway (ex: your only options are to use a single threaded app executor or a multithreaded app executor).
What are your thoughts on waiting for stageless before defining this api vs building this api (with stageless in mind)?

Reply:
I'm fine with closing this out until after stageless.

Reply:
> And with stageless on the horizon, this would need to be a global app setting anyway

This could be a per schedule setting in stageless.